docs/clusterfuzzlite.md

```diff
@@ -11,23 +11,38 @@ permalink: /
 ClusterFuzzLite makes [fuzzing] part of [Continuous Integration (CI)].
 ClusterFuzzLite is based on [ClusterFuzz].
 
-By fuzzing your code, ClusterFuzzLite can make it more secure as fuzzing is
-highly effective technique for finding bugs in software.
-
-ClusterFuzzLite supports [GitHub Actions] and [Google Cloud Build].
+Fuzzing is a highly effective technique for finding bugs in software.
+ClusterFuzzLite makes your code more secure by fuzzing changes before they even
+enter the codebase.
+ClusterFuzzLite supports [GitHub Actions] and [Google Cloud Build], getting the
+benefits of continuous fuzzing is simple if you can use these CI systems.
 Suppport for more CI systems is in-progress and
 [extending support to other CI systems] is easy.
 
-ClusterFuzzLite offers features that will make fuzzing an important part of your
-software development lifecycle, including:
+ClusterFuzzLite offers many useful features, including:
 1. Quickly fuzzing code changes (pull requests) before they land and allowing
-   users to download the crashing testcases.
-1. Asynchronous, long running fuzzing (batch fuzzing) that finds deeper bugs
-   missed during code change fuzzing.
+   you to download the crashing testcases.
+1. Continous longer running fuzzing (batch fuzzing) that asynchronously find
```

Comment:
Add "can", fix spelling on "Continuous"
`ContinUous longer running fuzzing (batch fuzzing) that CAN asynchronously find...`

Reply:
they can find bugs but they will always do so asynchronously so I think adding "can" makes it confusing and is overlycautious